src/Microsoft.AspNet.Identity.EntityFramework/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
```

Comment:
You should double check with @divega but I'm pretty sure EF needs these nav properties to be public get/set to do their proxy stuff...?